android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -434,12 +457,12 @@ public void onError(ErrorContainer errorContainer) {
     }
 
     private void purchaseProduct(final String productIdentifier,
-                                 final String type,
-                                 final String googleOldProductId,
-                                 @Nullable final Integer googleProrationMode,
-                                 @Nullable final Boolean googleIsPersonalizedPrice,
-                                 @Nullable final String presentedOfferingIdentifier,
-                                 final Result result) {
+            final String type,
```

Comment:
Hmm nitpick, but I think the indentation in these was correct following common java practices?

Reply:
Oh! Not sure how this happen 🤔 Undoing!